informal_update/fixtures/email_subscriptions.json

```diff
@@ -0,0 +1,23 @@
+[
+    {
+        "model": "informal_update.informalemailsubscriptions",
+        "pk": 1,
+        "fields": {
+            "share_with": "ifrc_secretariat"
+        }
+    },
+    {
+        "model": "informal_update.informalemailsubscriptions",
+        "pk": 2,
+        "fields": {
+            "share_with": "rcrc_network"
+        }
+    },
+    {
+        "model": "informal_update.informalemailsubscriptions",
+        "pk": 3,
+        "fields": {
+            "share_with": "rcrc_network_and_donors"
+        }
+    }
```

Comment:
Is this required here? We have custom logic in email send logic.

If not required, make sure to add a validation in models to never allow users to add this.

Reply:
No its not required. So i disabled add functionality in the admin for this model.